src/extras/mini-stats/mini-stats.js

```diff
@@ -14,6 +14,43 @@ import { Render2d } from './render2d.js';
  * @import { GraphicsDevice } from '../../platform/graphics/graphics-device.js'
  */
 
+/**
+ * @typedef {object} MiniStatsSizeOptions
+ * @property {number} width - Width of the graph area.
+ * @property {number} height - Height of the graph area.
+ * @property {number} spacing - Spacing between graphs.
+ * @property {boolean} graphs - Whether to show graphs.
+ */
+
+/**
+ * @typedef {object} MiniStatsGraphOptions
+ * @property {boolean} enabled - Whether to show the graph.
+ * @property {number} watermark - Watermark - shown as a line on the graph, useful for displaying a
+ * budget.
+ */
+
+/**
+ * @typedef {object} MiniStatsGenericOptions
```

Comment:
@mvaligursky I don't like the name here. Can you think of something better?

Reply:
maybe MiniStatsMetricOptions?